### DISCUSSION TITLE
"A boolean is being passed as a fourth parameter to window.open" when signing in

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#270] [Capstones!] [answered]
Gabby

Issue: I have set up google auth for my capstone. It is working correctly with users and admin, but I am getting the following error when clicking on the sign in button: "A boolean is being passed as a fourth parameter to window.open. This is not used and may cause an exception in a future release." The error is not preventing the app from working. I thought maybe it was because in the nav bar I've got the following logic: " if user is true then sign in else sign out" but it is exactly the same as what we've implemented in previous projects.

error: A boolean is being passed as a fourth parameter to window.open. This is not used and may cause an exception in a future release.

Google links: 
https://github.com/MicrosoftEdge/WebView2Feedback/issues/847
https://developer.mozilla.org/en-US/docs/Web/API/Window/open
https://www.w3schools.com/jsref/met_win_open.asp

I looked at the docs but did not really find an answer. w3schools mentions the "replace" optional parameter which is boolean, so maybe this is the boolean mentioned in the error? If it is, I am not sure how to go about fixing it. I checked previous projects and compared the nav bar, but I don't see anything different. In the sign in function (in auth), I had to add: 
``` 
provider.setCustomParameters({
    prompt: 'select_account'
  });
```
so that google auth would let me choose which email to use to sign in. I removed it and signed in again, but the error is still there. 

Code links: 
https://github.com/Gabrielle-Tobermann/Overture/blob/development/src/components/Navbar.js
https://github.com/Gabrielle-Tobermann/Overture/blob/development/src/helpers/auth.js

**Comment by ajawashington** [comment, marked as answer]
for future reference: https://github.com/firebase/firebase-js-sdk/issues/4988 

the solution still pending... 👀

**Reply by jimguistical** [reply]
Thank you, both!  Was running into this issue today.

**Reply by drteresavasquez** [reply]
Since we can make updates to Firebase, if you end up with issues, I would check the link @ajawashington posted for updates
